utils/canvas3.js

```
@@ -87,33 +85,43 @@ const CanvasOptions = {
+  scroll = null;
+  currentScroll = 0;
+  options = CanvasOptions;
+  animations = {
+    // cursorCallback: () => {},
+  };
+  footerGameBall = { x: 0, y: 0 };
+  mouse = { x: 0, y: 0, movementX: 0, movementY: 0, xPrev: 0, yPrev: 0 };
+
+  // triggerSectionPositions= {};
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)